src/Tokenizers/PHP.php

```diff
@@ -553,6 +553,23 @@ protected function tokenize($string)
                 $lastNotEmptyToken = ($newStackPtr - 1);
             }
 
+            /*
+                For Explicit Octal Notation prior to PHP 8.1 we need to combine the
+                T_NUMBER and T_STRING token values into a single token value, and
```

Comment:
Minor correction to comment: the token type is `T_LNUMBER` and not `T_NUMBER`

Reply:
IMO this code block also needs to be moved down after the initial comment processing bits and pieces.

I'm also wondering why the `$token` and `$tokens` variables are being adjusted and the loop is allowed to continue instead of the setting the token in the `$finalTokens` and `continue`-ing the loop.